core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -142,7 +139,7 @@ object Schema extends SchemaExtensions with SchemaMagnoliaDerivation with LowPri
   implicit val schemaForZonedDateTime: Schema[ZonedDateTime] = Schema(SDateTime)
   implicit val schemaForOffsetDateTime: Schema[OffsetDateTime] = Schema(SDateTime)
   implicit val schemaForDate: Schema[Date] = Schema(SDateTime)
-  implicit val schemaForLocalDateTime: Schema[LocalDateTime] = Schema(SDateTime)
+  implicit val schemaForLocalDateTime: Schema[LocalDateTime] = Schema(SString)
   implicit val schemaForLocalDate: Schema[LocalDate] = Schema(SDate)
```

Comment:
what about `LocalDate`? :)

Reply:
I would leave SDate schema for it, as swagger using it returns correct example, eg. "2017-07-21"